### DISCUSSION TITLE
Install minios. I can't execute bootinst.sh

### DISCUSSION THREAD

**Discussion by loyrloyr** [minios-linux/minios-live#38] [Q&A] [answered]
I have installed minios MINIMUM(64bit) on a key usb with the dd method. I like it but I need the persistence.
So I have installed it on a key usb formated in FAT32 and then double click bootinst.bat on Windows. It works but the result is strange about the persistence (14Go of free space on a key of 8Go ?) and I read that in FAT32 I can't have more than 4Go free space.
Therefore I try to install minios on a key formated in Ext4 but I can't succeed.
I copy the repertory minios on my key, then I can't execute bootinst.sh. I succeed apparently to make him executable (with chmod) but I can't run it and the terminal tells me : "bootinst.sh: 18: ./extlinux.x64: Permission denied Error installing boot loader."
Thank you for your help.

**Comment by FershoUno** [comment]
Hello, if you want to install MiniOS as ext4, I can suggest you the following instructions:

- install MiniOS using Fat32
- boot MiniOS in RAM (copy to ram option)
- use the built-in installer that MiniOS has and select the desired device with Ext4 file system and additionally if you want to have EFI support, then continue with the installation and that's it

**Reply by loyrloyr** [reply]
Thank you. Indeed, nice and simple idea.
Unfortunately I don't succeed to boot on ram (there is message "killed killed ..")
Is it "dangerous" or useless to try the built-in installer after a common boot ("default") ?

**Comment by crim50n** [comment, marked as answer]
To install on ext4, use `sudo bash ./bootinst.sh`

**Reply by loyrloyr** [reply]
Thank you for your help but it still doesn't work.

**Reply by dchurch315** [reply]
You need to make the appropriate file, extlinux.x64 in this case, executable before running the script.
